mne/viz/utils.py

```diff
@@ -726,11 +726,21 @@ def _plot_raw_onkey(event, params):
         params['ch_start'] -= params['n_channels']
         _channels_changed(params, len(params['inds']))
     elif event.key == 'right':
+        value = params['t_start'] + params['duration'] / 4
+        _plot_raw_time(value, params)
+        params['update_fun']()
+        params['plot_fun']()
+    elif event.key == 'shift+right':
```

Comment:
based on the output I pasted before:

https://github.com/mne-tools/mne-python/issues/4770#issuecomment-346375874

I don't think this will work for me. I'm on the recommended 3.6 env, so we'll probably need to do the shift-tracking workaround I mentioned before

Reply:
Why wouldn't this work? Based on your output, it should work.

Reply:
I print `event.key` in that snippet and it isn't ever `shift+right` or `shift+left`

Reply:
Oh, I was assuming you just pressed `shift`. The output didn't contain any arrow keys, could you please double-check what happens if you press (1) left and (2) shift+left?

Reply:
Ahh you're right, it works! I wonder why they made the arrow keys special -- maybe for this sort of use case

Reply:
actually I guess it's because for letters you get `A` vs `a` when you press shift

Reply:
Or because shift+a == A 😛